Lombiq.Tests.UI/Services/UITestExecutionSession.cs

```diff
@@ -305,13 +303,14 @@ private async Task CaptureAppSnapshotAsync(string dumpContainerPath)
             try
             {
                 var remotePath = appDumpPath;
-                if (_dockerConfiguration != null)
-                {
-                    appDumpPath = _dockerConfiguration.HostSnapshotPath;
-                    remotePath = _dockerConfiguration.ContainerSnapshotPath;
-                }
-
-                _sqlServerManager.TakeSnapshot(remotePath, appDumpPath, useCompressionIfAvailable: true);
+                var containerName = _dockerConfiguration?.ContainerName;
+                if (!string.IsNullOrEmpty(containerName)) remotePath = _dockerConfiguration.ContainerSnapshotPath;
+
+                await _sqlServerManager.TakeSnapshotAsync(
+                    remotePath,
+                    appDumpPath,
+                    containerName,
+                    useCompressionIfAvailable: true);
```

Comment:
How about a more readable version:

```suggestion
                var containerName = _dockerConfiguration?.ContainerName;
                var remotePath = string.IsNullOrEmpty(containerName)
                    ? appDumpPath
                    : _dockerConfiguration.ContainerSnapshotPath;

                await _sqlServerManager.TakeSnapshotAsync(
                    remotePath,
                    appDumpPath,
                    containerName,
                    useCompressionIfAvailable: true);
```
**Remark**: I'm just referring to the first 4 lines in comparison to your first 3 - the rest is me fighting GitHub.

Reply:
Just found [this occurrence](https://github.com/Lombiq/UI-Testing-Toolbox/pull/119/files#diff-7296bb52a59df5fa30fd8222aa55a27fb459709e7e0cb6539be92a6ac2606069R601) where you did exactly the same thing 😀